### PR TITLE
iganchev/angular templates

### LINKED ISSUE #20: Implement Financial Chart template

### PR DESCRIPTION
**Pull request by IvayloG**
Closes #20 .

### CODE REVIEW

File: templates/angular/ts/column-chart/default/files/src/app/components/__path__/__name__.component.ts [hunk 1]
```diff
@@ -0,0 +1,86 @@
+import { Component } from "@angular/core";
+import { ActivatedRoute, Routes } from "@angular/router";
```
Comment (damyanpetev) [inline templates/angular/ts/column-chart/default/files/src/app/components/__path__/__name__.component.ts]: These imports are not really needed for the component, right?

File: templates/angular/ts/column-chart/default/index.ts [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import { AngularTemplate } from "../../../../../lib/templates/AngularTemplate";
+
+class ColumnChartTemplate extends AngularTemplate {
+	constructor() {
+		super(__dirname);
+		this.components = ["Column Chart"];
+		this.controlGroup = "Charts";
+		this.listInComponentTemplates = true;
+		this.id = "column-chart";
+		this.framework = "angular";
```
Comment (damyanpetev) [inline templates/angular/ts/column-chart/default/index.ts]: I believe `AngularTemplate` already specifies `"angular"` framework by default.

File: templates/angular/ts/financial-chart/default/files/src/app/components/__path__/__name__.component.ts [hunk 3]
```diff
@@ -0,0 +1,152 @@
+import { Component } from "@angular/core";
+import { ActivatedRoute, Routes } from "@angular/router";
```
Comment (damyanpetev) [inline templates/angular/ts/financial-chart/default/files/src/app/components/__path__/__name__.component.ts]: Same as column.

File: templates/angular/ts/financial-chart/default/index.ts [hunk 4]
```diff
@@ -0,0 +1,17 @@
+import { AngularTemplate } from "../../../../../lib/templates/AngularTemplate";
+
+class FinancialChartTemplate extends AngularTemplate {
+	constructor() {
+		super(__dirname);
+		this.components = ["Financial Chart"];
+		this.controlGroup = "Charts";
+		this.listInComponentTemplates = true;
+		this.id = "financial-chart";
+		this.framework = "angular";
```
Comment (damyanpetev) [inline templates/angular/ts/financial-chart/default/index.ts]: Same :)

File: templates/angular/ts/line-chart/default/files/src/app/components/__path__/__name__.component.ts [hunk 5]
```diff
@@ -0,0 +1,110 @@
+import { Component } from "@angular/core";
+import { ActivatedRoute, Routes } from "@angular/router";
```
Comment (damyanpetev) [inline templates/angular/ts/line-chart/default/files/src/app/components/__path__/__name__.component.ts]: Marking this one as well.

File: templates/angular/ts/line-chart/default/index.ts [hunk 6]
```diff
@@ -0,0 +1,17 @@
+import { AngularTemplate } from "../../../../../lib/templates/AngularTemplate";
+
+class LineChartTemplate extends AngularTemplate {
+	constructor() {
+		super(__dirname);
+		this.components = ["Line Chart"];
+		this.controlGroup = "Charts";
+		this.listInComponentTemplates = true;
+		this.id = "line-chart";
+		this.framework = "angular";
```
Comment (damyanpetev) [inline templates/angular/ts/line-chart/default/index.ts]: ♻️

**Comment by IvayloG** [comment]
Closes #19 
Closes #18

File: templates/angular/ts/column-chart/default/files/src/app/components/__path__/__name__.component.ts (hunk 1 above)
Comment (IvayloG) [reply templates/angular/ts/column-chart/default/files/src/app/components/__path__/__name__.component.ts]: Removed...

File: templates/angular/ts/column-chart/default/index.ts (hunk 2 above)
Comment (IvayloG) [reply templates/angular/ts/column-chart/default/index.ts]: Removed...

**Pull request merged**